### PR TITLE
Seporate entity with document

### PR DESCRIPTION
**Pull request by IdoRoss**

### CODE REVIEW

File: libraroo-backend/src/auth/test/auth.service.spec.ts
Comment (zikili) [inline libraroo-backend/src/auth/test/auth.service.spec.ts:1]: not good enough

**Review by zikili — CHANGES_REQUESTED** [review]

File: libraroo-backend/src/catalog-books/catalog-books.repository.ts
Comment (zikili) [inline libraroo-backend/src/catalog-books/catalog-books.repository.ts:1]: Did you change the filterQuery from `filterQuery<CatalogBook>` to `filterQuery<CatalogBookDocument>` ?

Comment (IdoRoss) [reply libraroo-backend/src/catalog-books/catalog-books.repository.ts:1]: opsush

**Review by zikili — APPROVED** [review]
Very well done

**Pull request merged**